src/views/Mnemonic.js

```diff
@@ -2,16 +2,26 @@ import React, { useCallback, useEffect, useState } from 'react';
 import { Just, Nothing } from 'folktale/maybe';
 
 import View from 'components/View';
-import { MnemonicInput, PassphraseInput, HdPathInput } from 'components/Inputs';
+import {
+  MnemonicInput,
+  PassphraseInput,
+  HdPathInput,
+  useMnemonicInput,
+  usePassphraseInput,
+  useHdPathInput,
+} from 'components/Inputs';
 import { ForwardButton } from 'components/Buttons';
 
 import { ROUTE_NAMES } from 'lib/routeNames';
 import { useHistory } from 'store/history';
 import { walletFromMnemonic } from 'lib/wallet';
 import { useWallet } from 'store/wallet';
+import { Input } from 'indigo-react';
 
 export default function Mnemonic() {
```

Comment:
Changes here should apply to `/views/Login/mnemonic` instead, I think. Or both, both is good too.

Reply:
![https://media.giphy.com/media/hM9zK1qvsrwek/giphy.gif](https://media.giphy.com/media/hM9zK1qvsrwek/giphy.gif)